integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystemOpts.java

```diff
@@ -0,0 +1,278 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse;
+
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.PropertyKey;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * This class holds all fuse-related options used by libfuse and AlluxioFuse.
+ */
+public final class AlluxioFuseFileSystemOpts {
```

Comment:
Maybe better to separate them into groups for easier understanding.
e.g.
// core Alluxio Fuse Opts
  private final String mAlluxioPath;
  private final String mMountPoint;
  private final String mFsName;
// authentication related options
private final Class mFuseAuthPolicyClass;
private final Optional<String> mFuseAuthPolicyCustomGroup;
private final Optional<String> mFuseAuthPolicyCustomUser;
private final boolean mUserGroupTranslationEnabled;